assess/events/events.py

```diff
@@ -62,6 +62,17 @@ def events_from_process(cls, process):
         process_exit_dict["value"] = process_exit_dict["tme"] - \
             process_exit_dict["start_tme"]
 
+        if process_dict.get("pid", None) is None:
+            process_dict["pid"] = process_dict.get("node_id", None)
+            process_exit_dict["pid"] = process_dict.get("node_id", None)
+            if process_dict.get("_parent", None) is not None:
+                ppid = process_dict.get("_parent", None).node_id
```

Comment:
If the element is ``None``, getting ``.node_id`` would fail anyways. ``[]`` is faster in either case.
```suggestion
                ppid = process_dict["_parent"].node_id
```